core-services/prow/02_config/openshift/odo/_prowconfig.yaml

```diff
@@ -2,6 +2,18 @@ tide:
   merge_method:
     openshift/odo: squash
   queries:
+  - labels:
+    - approved
+    - kind
+    - lgtm
+    missingLabels:
```

Comment:
What does `missingLabels` do?

Reply:
I think it ensures that those labels are not there on the PR (hence "missing").

Reply:
From DPTP help desk:
missingLabels has "PR must not have any of these labels to merge" meaning
labels has "PR must have all these to merge" meaning